source/reference/minio-mc/mc-mirror.rst

```diff
@@ -15,26 +15,26 @@ Syntax
 
 .. start-mc-mirror-desc
 
-The :mc:`mc mirror` command synchronizes content to MinIO deployment, similar to
-the ``rsync`` utility. :mc:`mc mirror` supports filesystems, MinIO deployments,
-and other S3-compatible hosts as the synchronization source.
+The :mc:`mc mirror` command synchronizes content to MinIO deployment, similar to the ``rsync`` utility.
+:mc:`mc mirror` supports filesystems, MinIO deployments, and other S3-compatible hosts as the synchronization source.
 
 .. end-mc-mirror-desc
 
 .. tab-set::
 
    .. tab-item:: EXAMPLE
 
-      The following command synchronizes content from a local filesystem
-      directory to the ``mydata`` bucket on the ``myminio`` MinIO deployment.
+      The following command synchronizes content from a local filesystem directory to the ``mydata`` bucket on the ``myminio`` MinIO deployment.
 
       .. code-block:: shell
          :class: copyable
 
          mc mirror --watch ~/mydata myminio/mydata
 
-      The command "watches" the local filesystem and synchronizes any additional
-      operations to MinIO until explicitly terminated from the terminal:
+      The command "watches" for files added or removed on the local filesystem and synchronizes those operations to MinIO until explicitly terminated from the terminal. 
```

Comment:
Nit - terminated from the terminal is a little redundant (and if someone forked the process, that would require different steps)

It should be sufficient to state "until explicitly terminated"